server/lobbyconnection.py

```
@@ -1503,10 +1501,16 @@ async def abort_connection_if_banned(self):
-                )
-                raise BanError(ban_expiry, ban_reason)
+        ban_reason = row.reason
+        if now < ban_expiry:
+            self._logger.debug('Aborting connection of banned user: %s, %s, %s',
+                               self.player.id, self.player.login, self.session)
+            self.send_ban_message_and_abort(ban_expiry, ban_reason)
+
+    def send_ban_message_and_abort(self, ban_expiry, reason):
+        error_payload = {
+            "command": "banned",
+            "expires_at": ban_expiry.astimezone(timezone.utc).isoformat()
```

Ruff (0.16.3)

[error] Undefined name `timezone`

(F821)